Sources/MuxUploadSDK/Upload/ChunkedFileUploader.swift

```diff
@@ -76,11 +79,11 @@ class ChunkedFileUploader {
         cleanupResources()
         
         switch _currentState {
-        case .starting: fallthrough
-        case .uploading(_): do {
+        case .starting, .uploading:
             notifyStateFromMain(.canceled)
```

Comment:
`notifyStateFromMain` does not throw so putting it in a `do` clause is unnecessary